docker/lr-nanoplot/Makefile

```diff
@@ -0,0 +1,12 @@
+VERSION = 1.40.0-1
+TAG1 = us.gcr.io/broad-dsp-lrma/lr-nanoplot:$(VERSION)
```

Comment:
Not sure if it is needed but another [dockerfile](https://github.com/broadinstitute/long-read-pipelines/blob/main/docker/lr-gatk/Makefile) turned the name of the image into a variable 

```
IMAGE_NAME = lr-nanoplot
VERSION = 1.40.0-1

TAG1 = us.gcr.io/broad-dsp-lrma/$(IMAGE_NAME):$(VERSION)
TAG2 = us.gcr.io/broad-dsp-lrma/$(IMAGE_NAME):latest
```

Reply:
You are right. I should do this here too.